instrumentation/runtime/doc.go

```diff
@@ -12,21 +12,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// package runtime implements the conventional runtime metrics specified by OpenTelemetry.
+// package runtime implements two forms of runtime metrics for Golang OpenTelemetry users.
+//
+// The original implementation in this package uses ReadMemStats()
+// directly to report metric names.
 //
 // The metric events produced are:
-//   runtime.go.cgo.calls         -          Number of cgo calls made by the current process
-//   runtime.go.gc.count          -          Number of completed garbage collection cycles
-//   runtime.go.gc.pause_ns       (ns)       Amount of nanoseconds in GC stop-the-world pauses
-//   runtime.go.gc.pause_total_ns (ns)       Cumulative nanoseconds in GC stop-the-world pauses since the program started
-//   runtime.go.goroutines        -          Number of goroutines that currently exist
-//   runtime.go.lookups           -          Number of pointer lookups performed by the runtime
-//   runtime.go.mem.heap_alloc    (bytes)    Bytes of allocated heap objects
-//   runtime.go.mem.heap_idle     (bytes)    Bytes in idle (unused) spans
-//   runtime.go.mem.heap_inuse    (bytes)    Bytes in in-use spans
-//   runtime.go.mem.heap_objects  -          Number of allocated heap objects
-//   runtime.go.mem.heap_released (bytes)    Bytes of idle spans whose physical memory has been returned to the OS
-//   runtime.go.mem.heap_sys      (bytes)    Bytes of heap memory obtained from the OS
-//   runtime.go.mem.live_objects  -          Number of live objects is the number of cumulative Mallocs - Frees
-//   runtime.uptime               (ms)       Milliseconds since application was initialized
+//   process.runtime.go.cgo.calls         -          Number of cgo calls made by the current process
+//   process.runtime.go.gc.count          -          Number of completed garbage collection cycles
+//   process.runtime.go.gc.pause_ns       (ns)       Amount of nanoseconds in GC stop-the-world pauses
```

Comment:
Could we remove the _ns suffix (since it is implied by the unit already)?  In Prom, it would currently produce `process_runtime_go_gc_pause_ns_nanoseconds`.  Same comment for the total below.

Reply:
Well, this is the _OLD_ library's behavior and I don't want to touch it.